Improves the user manual (#4)
haddocking/haddock3-user-manual: src/modules/scoring.md

<hr>

## PRODIGY scoring modules

Two modules are using the PRODIGY methods for the evaluation of binding affinity.

> As this scoring is specific to either proteins or ligands, two modules are available, and should be used accordingly to the system you are working on:

Reviewer: ```suggestion
As this scoring is specific to either proteins or ligands, two modules are available, and should be used depending on which system you are working on:
```

<hr>

## PRODIGY scoring modules

Two modules are using the PRODIGY methods for the evaluation of binding affinity.
As this scoring is specific to either proteins or ligands, two modules are available, and should be used depending on which system you are working on:
- [`[prodigyprotein]`](#prodigyprotein-module): for the prediction of protein-protein binding affinities using PRODIGY
- [`[prodigyligand]`](#prodigyligand-module): for the prediction of protein-ligand binding affinities using PRODIGY-lig

### `[prodigyprotein]` module

Protein-protein binding affinity prediction using PRODIGY.

> This module performs scoring of protein-protein complexes using PRODIGY ([PyPI package](https://pypi.org/project/prodigy-prot/)).

Reviewer 2: would rather indicate the github repo then pypi here and on lig

> Note that this approach is limitted to protein-protein intreactions containing standard amino-acids, and any usage outside of this scope will results in failures.

Reviewer: ```suggestion
Note that this approach is limited to protein-protein interactions containing standard amino-acids.
```

> For a detailed explanation about the PRODIGY method can be found in published research articles:

Reviewer: ```suggestion
A detailed explanation about PRODIGY can be found in published research articles:
```

  - ["A", "B"]      => Contacts calculated (only) between chains A and B.
  - ["A,B", "C"]    => Contacts calculated (only) between chains A and C; and B and C.
  - ["A", "B", "C"] => Contacts calculated (only) between chains A and B; B and C; and A and C.
- `to_pkd`: Converts predicted binding affinity values to pKd values.

More information about `[prodigyprotein]` parameters can be accessed [here](https://bonvinlab.org/haddock3/modules/scoring/haddock.modules.scoring.prodigyprotein.html#default-parameters) or retrieved by running:

```bash
haddock3-cfg -m prodigyprotein
```

### `[prodigyligand]` module

This module performs binding affinity prediction of protein-ligand complexes using PRODIGY-lig ([GitHub](https://github.com/haddocking/prodigy-lig), [PyPI package](https://pypi.org/project/prodigy-lig/)).

> For a detailed explanation about the PRODIGY-lig method can be found in published research articles:

Reviewer: ```suggestion
A detailed explanation about PRODIGY-lig can be found in published research articles:
```
